concrete-core-fixture/src/generation/prototyping/lwe_ciphertext.rs

```diff
@@ -48,6 +49,11 @@ pub trait PrototypesLweCiphertext<
         &mut self,
         ciphertext: &Self::LweCiphertextProto,
     ) -> Self::PlaintextProto;
+    fn transform_raw_to_ciphertext(&mut self, raw: &[Precision::Raw]) -> Self::LweCiphertextProto;
+    fn transform_ciphertext_to_raw(
```

Comment:
Here, to be consistent with the rest of the naming, it should be `raw_vec`.

Reply:
will do

Reply:
fixed for lwe and glwe